index.html

```diff
@@ -111,8 +112,65 @@ <h3 class="card-heading">Multi-Post Stories Gain + Glory</h3>
   </section>
   <!-- About section  -->
   <section id="about">
+    <img src="assets/images/ab1.png" alt="" id="about-img-1">
+    <h2 class="subheading" id="about-title">About Me</h2>
+    <p id="about-content"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
```

Comment:
- [ ] Please make sure that the distance between the about myself title and paragraph  matches the design 
![image](https://user-images.githubusercontent.com/74744787/235237531-1f6cb54e-a5c1-4f02-a633-03d3264a5671.png)
Figma 
![image](https://user-images.githubusercontent.com/74744787/235237596-5475d14f-5774-4d83-9db9-548c29df2136.png)